iib/workers/tasks/build.py

```diff
@@ -620,20 +626,24 @@ def handle_add_request(
 
     set_request_state(request_id, 'in_progress', 'Creating the manifest list')
     output_pull_spec = _create_and_push_manifest_list(request_id, arches)
-
-    if legacy_support_packages:
-        export_legacy_packages(
-            legacy_support_packages, request_id, output_pull_spec, cnr_token, organization
-        )
-
-    _finish_request_post_build(
+    modified_index_image = _update_index_image_pull_spec(
         output_pull_spec,
         request_id,
         arches,
         from_index,
         overwrite_from_index,
         overwrite_from_index_token,
     )
+    if legacy_support_packages:
+        export_legacy_packages(
+            legacy_support_packages, request_id, modified_index_image, cnr_token, organization
+        )
+    set_request_state(
+        request_id,
+        'complete',
+        'Operator bundle(s) added to index image, \
+                                               completing the request successfully',
```

Comment:
Wait! Python black isn't complaining to this formatting? That's strange. Does `tox` work successfully?

Reply:
It was a suggestion from black, unexpected for me too

Reply:
I guess I mixed up 2 suggestions